dymos/examples/min_time_climb/test/test_min_time_climb_simulate_failure.py

```diff
@@ -95,9 +95,9 @@ def test_simulate_raises_analysis_error(self):
 
         with self.assertRaises(om.AnalysisError) as e:
             dm.run_problem(p, run_driver=False, simulate=True)
-        self.assertEqual(str(e.exception), "simulation of traj.phases.phase0.segments.segment_2 "
-                                           "failed: Required step size is less than spacing "
-                                           "between numbers.")
+        self.assertEqual(str(e.exception), "SegmentSimulationComp (traj.phases.phase0.segments.segment_2): Error"
```

Comment:
OpenMDAO is already giving us the component in which the error was raised.  Can we please omit it from our error message so it's not redundant?

Reply:
The assertEqual will fail if that text is changed.

Reply:
It's ok to change the test if we change the error message.